sql/sys_vars.cc

```diff
@@ -1350,16 +1350,10 @@ static Sys_var_mybool Sys_large_files_support(
        READ_ONLY GLOBAL_VAR(opt_large_files),
        CMD_LINE_HELP_ONLY, DEFAULT(sizeof(my_off_t) > 4));
 
-static Sys_var_uint Sys_large_page_size(
-       "large_page_size",
-       "If large page support is enabled, this shows the size of memory pages",
-       READ_ONLY GLOBAL_VAR(opt_large_page_size), NO_CMD_LINE,
-       VALID_RANGE(0, UINT_MAX), DEFAULT(0), BLOCK_SIZE(1));
```

Comment:
We can't remove it without marking it deprecated first. See how `DEPRECATED()` is used in the sys_vars.cc. Alternatively you can remove it but add to the `removed_opts` of mysqld.cc.